data/discord

```diff
@@ -21,6 +24,8 @@ the Minecraft Cursed Legacy discord
 the FabLabs discord
 the Furbic discord
```

Comment:
there’s a furbric discord now? :help_me:

Reply:
Idk, didn't add that one :p